main/conceptual/js-programming/hardened-js.md

```diff
@@ -51,32 +49,32 @@ uses a proposed syntax for [eventual send](./eventual-send.md),
 which we will get to soon.
 :::
 
-## Object-capabilities (ocaps)
+## Object Capabilities (ocaps)
```

Comment:
It's capitalized OCap / OCaps in places like https://agoric.com/wp-content/uploads/2022/08/Agoric-White-Paper-v1.1.pdf

Reply:
This is probably going to be more information than you want, but....
I looked online to see how OCap's capitalization is usually handled, and there doesn't seem to be a consensus. When it's spelled out, it's usually (but not always) not hyphenated or capitalized, but it gets capitalized here because it's a section header. As for "OCap", its capitalization is all over the place, so I generally left it as whatever we had it as. I really should decide on our official style guide for it, and then make the docs consistent throughout, but I have higher priority stuff to worry about. So for now, we're going with "willy-nilly capitalization".